examples/llm/main.py

```diff
@@ -86,7 +86,8 @@ def main(cfg):
     )
 
     reproducibility.seed_all(cfg.seed)
-    dist.initialize_dist(get_device(None))
+    dist.initialize_dist(get_device(None),
+                         timeout=cfg.get('dist_timeout', 1500.0))
```

Comment:
1500 is a pretty random default (25 min)
300 is used somewhere in composer (5 min)
1800 is the torch dist default time out (30 min)

Reply:
done